lib/puppet/defaults.rb

```diff
@@ -791,7 +791,7 @@ def self.default_diffargs
       :default => ['md5', 'sha256'],
       :type    => :array,
       :desc    => 'Checksum types supported by this agent for use in file resources of a static
-        catalog. Valid types are md5, md5lite, sha256, sha256lite, sha1, sha1lite, mtime, ctime.',
+        catalog. Values must be comma-separated. Valid types are md5, md5lite, sha256, sha256lite, sha1, sha1lite, mtime, ctime.',
```

Comment:
Doesn't follow line-wrapping for descriptions elsewhere in the file. Does it matter?